tests/pytests/test_config.py

```diff
@@ -36,8 +36,8 @@ def testConfig(env):
 @skip(cluster=True)
 def testConfigErrors(env):
     env.expect(config_cmd(), 'set', 'MINPREFIX', 1, 2).equal('EXCESSARGS')
-    env.expect(config_cmd(), 'no_such_command', 'idx').equal('No such configuration action')
-    env.expect(config_cmd(), 'idx').error().contains('wrong number of arguments')
+    env.expect(config_cmd(), 'no_such_command', 'idx').equal(f'unknown subcommand \'no_such_command\'. Try {config_cmd()} HELP.')
```

Comment:
Error `No such configuration action` cannot occur anymore, right? (will fail by Redis before getting to the module)

Reply:
correct, not sure if this is a breaking change or not.